src/mouse.cpp

```diff
@@ -27,7 +27,7 @@ static Client*        g_win_drag_client = nullptr;
 static Monitor*       g_drag_monitor = nullptr;
 static MouseDragFunction g_drag_function = nullptr;
 
-static Cursor g_cursor;
+Cursor g_cursor;
```

Comment:
Please make this a member of MouseManager. `g_cursor` is only used in `mouse_init` and `mouse_destroy`.

Reply:
I wanted to do as little as possible along with the introduction of the class, that's why I left it. But you're right, moving it inside is better than making it temporarily global.